process/sync/shardblock_test.go

```diff
@@ -2135,10 +2136,17 @@ func TestBootstrap_ForkChoiceIsEmptyCallRollBackOkValsShouldWork(t *testing.T) {
 		},
 	}
 
-	hasher := &mock.HasherMock{}
+	hasher := &mock.HasherStub{
+		ComputeCalled: func(s string) []byte {
+			return currentHdrHash
+		},
+	}
 
 	//a marshalizer stub
 	marshalizer := &mock.MarshalizerStub{
+		MarshalCalled: func(obj interface{}) ([]byte, error) {
+			return json.Marshal(obj)
```

Comment:
You are using the gin implementation. Works but you'd better make use of the golang's impl.

Reply:
Removed